testing_and_setup/seaice/configurations/snicar_shortwave/streams.seaice

```diff
@@ -0,0 +1,458 @@
+        <var name="uOceanVelocityVertexGeo"/>
+        <var name="vOceanVelocityVertexGeo"/>
+        <var name="freezingMeltingPotential"/>
+        <var name="shortwaveScalingFactor"/>
+        <var name="airTemperature"/>
+        <var name="congelation"/>
+        <var name="frazilFormation"/>
+        <var name="snowiceFormation"/>
+        <var name="snowMelt"/>
+        <var name="surfaceIceMelt"/>
+        <var name="basalIceMelt"/>
+        <var name="lateralIceMelt"/>
+        <var name="iceVolumeTendencyThermodynamics"/>
+        <var name="iceVolumeTendencyTransport"/>
+        <var name="iceAreaTendencyThermodynamics"/>
+        <var name="iceAreaTendencyTransport"/>
+        <var name="iceAgeTendencyThermodynamics"/>
+        <var name="iceAgeTendencyTransport"/>
+        <var name="iceAgeCell"/>
+        <var name="firstYearIceAreaCell"/>
+        <var name="levelIceAreaCell"/>
+        <var name="levelIceVolumeCell"/>
+        <var name="ridgedIceAreaAverage"/>
+        <var name="ridgedIceVolumeAverage"/>
+        <var name="bulkSalinity"/>
+        <var name="broadbandAlbedo"/>
+        <var name="absorbedShortwaveFluxInitialArea"/>
+        <var name="latentHeatFluxInitialArea"/>
+        <var name="sensibleHeatFluxInitialArea"/>
+        <var name="longwaveUpInitialArea"/>
+        <var name="evaporativeWaterFluxInitialArea"/>
+        <var name="meltPondAreaFinalArea"/>
+        <var name="meltPondDepthFinalArea"/>
+        <var name="meltPondLidThicknessFinalArea"/>
+</stream>
+
+<immutable_stream name="LYqSixHourlyForcing"
+                  type="input"
+                  filename_template="forcing/atmosphere_forcing_six_hourly.$Y.nc"
+                  filename_interval="0001-00-00_00:00:00"
+		  reference_time="2000-01-01_03:00:00"
+                  input_interval="none" />
+
+<immutable_stream name="LYqMonthlyForcing"
+                  type="input"
+                  filename_template="forcing/atmosphere_forcing_monthly.nc"
+                  filename_interval="none"
+                  input_interval="none" />
+
+<immutable_stream name="NCARMonthlySSTForcing"
+                  type="input"
+                  filename_template="forcing/ocean_forcing_monthly.nc"
+                  filename_interval="none"
+                  input_interval="none" />
+
+<immutable_stream name="NCARMonthlyForcing"
+                  type="input"
+                  filename_template="forcing/ocean_forcing_monthly.nc"
+                  filename_interval="none"
+                  input_interval="none" />
+
+<immutable_stream name="StandardAerosolsInput"
+                  type="input"
+                  filename_template="forcing/standard_optics_mpas_seaice.nc"
+                  filename_interval="none"
+                  input_interval="initial_only" />
+
+<immutable_stream name="SnicarInput"
+                  type="input"
+                  filename_template="forcing/snicar_optics_4bnd_snow_and_aerosols.nc"
```

Comment:
Should not this be snicar_optics_5bnd_snow_and_aerosols.nc? Currently with the files I've been provided with this configuration fails because of this misnaming. Please test additions to PR.

Reply:
You're right. Thanks for catching this typo. It is fixed.